foundation-ui/mount/Mount.js

```diff
@@ -0,0 +1,108 @@
+import React, {PropTypes} from 'react';
+
+import {MountService} from './index';
+import {CHANGE} from './MountEvent';
```

Comment:
I named those `EventTypes` in both routing and navigation services, just saying :)